src/Symfony/Component/HttpFoundation/Session/Storage/MockArraySessionStorage.php

```diff
@@ -75,8 +73,6 @@ public function __construct($name = 'MOCKSESSID', MetadataBag $metaBag = null)
     }
 
     /**
-     * Sets the session data.
```

Comment:
> But then the @param below can be removed as well...

Yep, I'd removed it.

> and thousands of other phpdocs. 

Probably, yeah.

> A good time will be when we can really get rid of the vast majority of current phpdocs (but on all the code base at once).

I guess, with php7 and scalar typehinting?

> I'm 👎 for these changes.

You mean this one change or other changes as well? I can rewrite the PR of course.